changelogs/fragments/oc-migration-removal.yml

```diff
@@ -0,0 +1,14 @@
+removed_features:
+- |
+  The ``oc`` connection plugin has been removed from this collection.
+  It has been migrated to the `community.okd <https://galaxy.ansible.com/community/okd>`_ collection.
+  If you use ansible-base 2.10 or newer, redirections have been provided.
+
+  If you use Ansible 2.9 and installed this collection, you need to adjust the FQCNs (``community.general.oc`` → ``community.okd.oc``) and make sure to install the community.okd collection.
+breaking_changes:
+- |
+  If you use Ansible 2.9 and the ``oc`` connection plugin from this collections, community.general 2.0.0 results in errors when trying to use the oc content by FQCN, like ``community.general.oc``.
+  Since Ansible 2.9 is not able to use redirections, you will have to adjust your playbooks and roles manually to use the new FQCNs (``community.okd.oc`` for the previous example) and to make sure that you have ``community.okd`` installed.
```

Comment:
```suggestion
  Since Ansible 2.9 is not able to use redirections, you will have to adjust your playbooks and roles manually to use the new FQCNs (``community.okd.oc``) and to make sure that you have ``community.okd`` installed.
```

Reply:
I extended the text a bit since this is a single connection plugin, and the text was more optimized for a group of plugins out of which most are modules.